importer-declarative.md

```diff
@@ -0,0 +1,164 @@
+# Declarative Transformations
+
+A declarative transformation is a JSON object that describes how a site should be imported to AEM Edge Delivery without requiring any code to be written. 
+
+## JSON Structure
+
+```json
+{
+  "root": "<string>",
+  "cleanup": {
+    "start": ["<string>"]
+  },
+  "blocks": [
+    {
+      "type": "<block name>",
+      "target": "append | replace | prepend",
```

Comment:
`target` - My mind goes to `event.target` for example - a DOM element.  Should this be an `action` or `mode` type of name?

Reply:
I didn't love this name either so will come up with something better.

Reply:
`insertMode`?